rpm/libhybris.spec

```diff
@@ -1,3 +1,6 @@
+%define USE_MALI %{getenv:USE_MALI}
+%define USE_EXPEREMENTAL %{getenv:USE_EXPEREMENTAL}
```

Comment:
Words such as 'experimental' have been renamed to reflect of what they actually do instead:
https://github.com/libhybris/libhybris/pull/360/commits/a3213117c56eabd462cbc225ae65ad0bb99af565